src/frontend/ai-reviewer-admin/src/domain/documents/DocumentList.js

```diff
@@ -15,7 +15,7 @@ export default function DocumentList({ configurations }) {
         <span className="d-none d-sm-inline col-sm-3">Action</span>
       </div>
       <ul>
-        {configurations.map((configuration) => (
+        {configurations && configurations.map && configurations.map((configuration) => (
```

Comment:
I actually changed this in my PR.  I moved this logic to the service layer to throw an error if the result is not an actual map, in which case, this configurations object will either be null or a valid object.  Hopefully. :)